topology/Academia Sinica/ASGC/ASGC-IGWN.yaml

```
@@ -0,0 +1,34 @@
+Production: true
+SupportCenter: Self Supported
+GroupDescription: This is an htcondor-ce cluster for the IGWN service on ASGC.
```

```suggestion
GroupDescription: These are resources for IGWN running at the Academia Sinica Grid Computing Center
```

This is phrased oddly. The HTCondor-CE dumps pilots into their local batch scheduler so it'd be more correct to say that the CE is serving the cluster. However, it's a group so they could feasibly run multiple services so highlighting the CE also isn't quite right.

IGWN is also the collaboration of LIGO/VIRGO so they're not really a service and ASGC is the computing center